docs/content/stable/drivers-orms/smart-drivers-ycql.md

```diff
@@ -0,0 +1,107 @@
+| [YugabyteDB Ruby Driver for YCQL](https://github.com/yugabyte/cassandra-ruby-driver) | [DataStax Ruby Driver](https://github.com/datastax/ruby-driver) | [Documentation](../rust/yb-rust-postgres) |
+
+All YugabyteDB smart driver libraries are actively maintained, and receive bug fixes, performance enhancements, and security patches.
+
+## Key features
+
+YugabyteDB smart drivers have the following key features.
+
+| Feature | Notes |
+| :--- | :--- |
+| [Retry policy](#retry-policy) | Smart driver retries certain operations in case of failures. |
+| [Multiple contact points](#multiple-contact-points) | Smart driver supports providing multiple contact points to ensure that the application initializes successfully, even if one or few of the contact points are inaccessible. |
+| [Partition-aware load balancing policy](#partition-aware-load-balance-policy) (YugabyteDB-specific) | Smart driver can target a specific tablet/node for a particular query where the required data is hosted. |
+| [JSONB support](#jsonb-support) (YugabyteDB-specific) | Smart driver supports the use of JSONB data type for the table columns. |
+
+## Overview
+
+YugabyteDB is a distributed, fault tolerant, and highly available database with low latencies for reads and writes. Data in YugabyteDB is automatically sharded, replicated, and balanced across multiple nodes that can potentially be in different availability zones and regions. For better performance and fault tolerance, you can also balance application traffic (database connections) across the nodes in the universe to avoid excessive CPU and memory load on any single node.
+
+### Retry policy
+
+The smart driver can retry certain operations if they fail the first time, and this is governed by a retry-policy (the default number of retries being one for a failed operation for certain cases depending on the language driver). The retry may happen on the same node or the next available node as per the query's plan.
+
+Some of the drivers allow you to provide a custom retry policy.
+Refer to [Retries](https://docs.datastax.com/en/developer/java-driver/4.15/manual/core/retries/#retries) in DataStax documentation for information on built-in retry polices.
+
+### Multiple contact points
+
+Usually, the contact point is the host address given in the connection configuration by the application. The driver creates a control connection to this host address. If this host address is unavailable at the time of application's initialization, the application itself fails.
+To avoid such a scenario, you can specify comma-separated multiple host addresses for your applications as contact points. If the first host is not accessible, the driver tries to create a control connection to the next contact point, and so on, until it succeeds or all contact points are attempted.
```

Comment:
```suggestion
To avoid this scenario, you can specify multiple comma-separated host addresses for your applications as contact points. If the first host is not accessible, the driver tries to create a control connection to the next contact point, and so on, until it succeeds or all contact points are attempted.
```